modin/pandas/io.py

```diff
@@ -884,7 +884,7 @@ def return_handler(*args, **kwargs):
 
 
 @_inherit_docstrings(pandas.ExcelFile)
-class ExcelFile(ClassLogger, pandas.ExcelFile):  # noqa: PR01, D200
+class ExcelFile(ClassLogger, pandas.ExcelFile, modin_layer="PANDAS-IO"):  # noqa: PR01, D200
```

Comment:
```suggestion
class ExcelFile(ClassLogger, pandas.ExcelFile):  # noqa: PR01, D200
```
This is also pandas API.